src/utils/tools/svgToPng.ts

```diff
@@ -0,0 +1,18 @@
+import pify from 'pify'
+import svg2img from 'svg2img'
+const psvg2img = pify(svg2img)
+
+async function svgToPng (pngSize: string, svgPath: string) {
+  try {
+    const svgOptions = {
+      width: pngSize,
+      height: pngSize
+    }
+    const buffer = await psvg2img(svgPath, svgOptions)
+    return buffer
+  } catch (err) {
+    throw new Error(`An error occured while converting the svg to png: ${err}`)
+  }
+}
+
```

Comment:
This is the same `svgToPng` utility: 

```JS
const fs = require('fs');
const svg2img = require('svg2img');
const tempDir = `${__dirname}/../temp/`;

convertSvgToPng = (iconName, pngSize, theme = 'svg') => {
  return new Promise((resolve, reject) => {
    if (!fs.existsSync(tempDir)) {
      fs.mkdirSync(tempDir);
    }
    svg2img(`${__dirname}/../${theme}/${iconName}.svg`, { width: pngSize, height: pngSize }, (error, buffer) => {
      if (error) {
        return reject();
      }
      fs.writeFileSync(`${__dirname}/../temp/${iconName}_${pngSize}.png`, buffer);
      resolve();
    });
  });
};

module.exports = convertSvgToPng;

```

The changes: 
1) Using a promise style version (converted by `pify`). 
2) Passing the `svgPath` as parameter to adapt any changes easily. 
3) Making it only responsible about converting the `SVG` and return the buffer, the write process  will be called from the controllers.